change-notes/1.18/analysis-csharp.md

```diff
@@ -15,6 +15,7 @@
 
 | **Query**                   | **Tags**  | **Purpose**                                                        |
 |-----------------------------|-----------|--------------------------------------------------------------------|
+| Arbitrary file write during zip extraction ("ZipSlip") | security external/cwe/cwe-022  | Identifies zip extraction routines which allow tainted path and arbitrary file overwrite vulnerabilities.
```

Comment:
- "ZipSlip" > "Zip Slip" (According to the linked reference: https://snyk.io/research/zip-slip-vulnerability)
- It's useful to add the query ID in parentheses: (cs/zipslip)
- The query purpose is slightly unclear to me: "tainted _paths_" (plural)? (Or does this refer to "tainted path vulnerabilities"?)

Reply:
I've changed ZipSlip to Zip Slip, and added the query ID.

I've removed the bit about tainted paths, so that it now reads:
"Identifies zip extraction routines which allow arbitrary file overwrite vulnerabilities."